### DISCUSSION TITLE
How to see log for all incoming and outgoing connection on rustdesk server

### DISCUSSION THREAD

**Discussion by vyncusp** [rustdesk/rustdesk-server#497] [Q&A]
Hi,

Any idea how to see log for all incoming and outgoing connection on rustdesk server?

Example:
incoming to rustdesk id 1000 from rusdesk id 300 on 27 Apr 2023 07:00 until 27 Apr 2023 07:30
incoming to rustdesk id 2100 from rusdesk id 500 on 27 Apr 2023 09:00 until 27 Apr 2023 09:22

Where can i get these kind on information?

**Comment by sergiovilchesp** [comment]
do you find the logs? @vyncusp

**Reply by vyncusp** [reply]
Still no luck :(

**Reply by Jaynornj** [reply]
Hey, did you find a way for this?

**Reply by Grelo4ka** [reply]
Would be cool to see that

**Comment by DevJupyHUB** [comment]
This is how we do it:

ELK + (y)our own Grok pattern + with Filebeat on RustDesk server.
ELK stack receives Filebeat data + pipeline: "logs-rustdesk-server-one-processor"

![image](https://github.com/user-attachments/assets/9d9e8e11-4c61-4494-8bd4-d81120471615)


_pre-processor:_

![image](https://github.com/user-attachments/assets/f0941cac-99fd-46dd-8475-1affb4e273d8)


_**RAW data in Kibana, like this:**_

![image](https://github.com/user-attachments/assets/880c0b7d-3768-4345-8f65-86c44fb5271d)


In the end you can make the data into any Kibana alerts &/or DASH you want.
